### PR TITLE
Add group_member_post.go

### PR DESCRIPTION
**Pull request by S-H-GAMELINKS**
Hi, Thanks for this great library.

I added support for the following Group APIs:
- Add group member https://marketplace.zoom.us/docs/api-reference/zoom-api/groups/groupmemberscreate

Inaddtional, add group member exsample.

### CODE REVIEW

File: member_type.go [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package zoom
+
+// Member represents an group member
+type Member struct {
+	ID        string `json:"id"`
+	Email     string `json:"email"`
+	FirstName string `json:"first_name"`
+	LastName  string `json:"last_name"`
+	Type      int64  `json:"type"`
+	AddedAt   string `json:"added_at"`
```
Comment (rafecolton) [inline member_type.go:10]: According to the api doc you linked, the only fields populated here are ID and email - where are you getting the other fields from?

File: group_member_post.go [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+// AddMembers calls POST /groups/{groupId}/members
+func AddMembers(opts AddMemberOptions) (Member, error) {
+	return defaultClient.AddMembers(opts)
+}
+
+// AddMembers calls POST /groups/{groupId}/members
+// https://marketplace.zoom.us/docs/api-reference/zoom-api/groups/groupmemberscreate
+func (c *Client) AddMembers(opts AddMemberOptions) (Member, error) {
+	var ret = Member{}
```
Comment (rafecolton) [inline group_member_post.go:28]: I think this should return `[]Member` not `Member`, right?

File: member_type.go (hunk 1 above)
Comment (S-H-GAMELINKS) [reply member_type.go:10]: Sorry, I'm referenced other api doc. Fix it.

File: group_member_post.go (hunk 2 above)
Comment (S-H-GAMELINKS) [reply group_member_post.go:28]: Sorry, this func should return theese response.

```go
type ResopnseAddGroupMembers struct {
	Ids		 string `json:"ids"`
	AddedAt  string `json:"added_at"`
}
```

See this api doc response.
ref: https://marketplace.zoom.us/docs/api-reference/zoom-api/groups/groupmemberscreate

File: member_type.go [hunk 3]
```diff
@@ -0,0 +1,12 @@
+package zoom
+
+// Member represents an group member
+type Member struct {
+	ID        string `json:"id"`
+	Email     string `json:"email"`
+}
+
+type ResopnseAddGroupMembers struct {
+	Ids		 string `json:"ids"`
```
Comment (rafecolton) [inline member_type.go:10]:
```suggestion
	IDs		 string `json:"ids"`
```

Comment (rafecolton) [reply member_type.go:10]: The linter is going to complain, this struct needs a comment. The Zoom docs don't say what format IDs takes - is it comma-delimited? Could you please do a test where you make this API call against a real account to add multiple members and include in your comment an example of what this data looks like?

**Comment by rafecolton** [comment]
Looks like you have some minor formatting issues as well. This is almost there though, thanks for submitting!

File: member_type.go (hunk 3 above)
Comment (S-H-GAMELINKS) [reply member_type.go]: API return string has comma-delimited.
I'll add these comment!

File: member_type.go [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+// Member represents an group member
+type Member struct {
+	ID    string `json:"id"`
+	Email string `json:"email"`
+}
+
+// ResopnseAddGroupMembers represents response for added member to group
+type ResopnseAddGroupMembers struct {
+	// Ids has comma-delimited, like 'xxxxxxxxxx, xxxxxxxxxx'
+	Ids     string `json:"ids"`
```
Comment (rafecolton) [inline member_type.go:12]: Can you please change `Ids` to `IDs`? Also can you confirm that there is a space between the comma-delimited values for member IDs?

Comment (S-H-GAMELINKS) [reply member_type.go:12]: OK, I'll change it.
And, there has no space, remove it.

File: group_member_post.go [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+import "fmt"
+
+// AddMenbersPath - v2 path for add group members
+const AddMenbersPath = "/groups/%s/members"
+
+// AddMemberInfo are details about a member
+type AddMemberInfo struct {
+	ID    string `json:"id"`
+	Email string `json:"email"`
+}
```
Comment (rafecolton) [inline group_member_post.go:12]: It looks like this type is unused?

Comment (S-H-GAMELINKS) [reply group_member_post.go:12]: Yes, unneeded.

File: member_type.go [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+	ID    string `json:"id"`
+	Email string `json:"email"`
+}
+
+// ResopnseAddGroupMembers represents response for added member to group
+type ResopnseAddGroupMembers struct {
+	// IDs has comma-delimited, like 'xxxxxxxxxx,xxxxxxxxxx'
+	IDs     string `json:"ids"`
+	AddedAt string `json:"added_at"`
+}
```
Comment (rafecolton) [inline member_type.go:14]: Are either of these types used in other API calls? If so, they can remain in this separate file, but if not, let's move them into group_member_post.go

Comment (S-H-GAMELINKS) [reply member_type.go:14]: No used, I'll move to `group_member_post.go`

**Review by rafecolton — APPROVED** [review]
LGTM, thanks for the contribution!

**Comment by rafecolton** [comment]
Looks like your commits are not signed. Can you please sign your commits, and then I will merge?

**Comment by S-H-GAMELINKS** [comment]
OK, signed commit push

**Review by rafecolton — APPROVED** [review]

**Comment by S-H-GAMELINKS** [comment]
Thanks for marged!

**Comment by rafecolton** [comment]
You're welcome, thanks for the contribution!

**Pull request merged**